test/verify/check-storage-anaconda

```diff
@@ -0,0 +1,121 @@
+        self.dialog({})
+
+        # Create an encrypted filesystem
+        self.click_dropdown(self.card_row("Storage", name="lvol0"), "Format")
+        self.dialog_wait_open()
+        self.dialog_wait_val("at_boot", "local")
+        self.dialog_set_val("mount_point", "/")
+        self.dialog_set_val("type", "ext4")
+        self.dialog_set_val("crypto", self.default_crypto_type)
+        self.dialog_set_val("passphrase", "vainu-reku-toma-rolle-kaja")
+        self.dialog_set_val("passphrase2", "vainu-reku-toma-rolle-kaja")
+        self.dialog_apply()
+        self.dialog_wait_close()
+
+        self.assertNotIn("noauto", m.execute("findmnt --fstab -n -o OPTIONS /sysroot"))
+
+        b.wait_visible(self.card_row_col("Storage", 3, 5) + " .usage-bar")
+        b.assert_pixels("body", "page")
+
+        # Unount/mount the filesystem, edit mount options
+        self.click_card_row("Storage", location="/")
+        b.wait_visible(self.card("Encryption"))
+        b.wait_in_text(self.card_desc("ext4 filesystem", "Mount point"), "/ (stop boot on failure)")
+        b.click(self.card_desc("ext4 filesystem", "Mount point") + " button")
+        self.dialog_wait_open()
+        self.dialog_wait_val("mount_point", "/")
+        self.dialog_set_val("mount_options.ro", val=True)
+        self.dialog_apply()
+        self.dialog_wait_close()
+        self.assertIn("ro", m.execute("findmnt --fstab -n -o OPTIONS /sysroot"))
+        b.click(self.card_button("ext4 filesystem", "Unmount"))
+        self.dialog_wait_open()
+        b.wait_text("#dialog .pf-v5-c-modal-box__title-text", "Unmount filesystem /")
+        self.dialog_apply()
+        self.dialog_wait_close()
+        self.wait_not_mounted("Filesystem")
+        self.assertIn("noauto", m.execute("findmnt --fstab -n -o OPTIONS /sysroot"))
+        b.click(self.card_button("Filesystem", "Mount"))
+        self.dialog_wait_open()
+        self.dialog_wait_val("mount_point", "/")
+        self.dialog_set_val("passphrase", "vainu-reku-toma-rolle-kaja")
+        self.dialog_apply()
+        self.dialog_wait_close()
+        self.wait_mounted("ext4 filesystem")
+        self.assertNotIn("noauto", m.execute("findmnt --fstab -n -o OPTIONS /sysroot"))
+
+        # Check and delete volume group
+        b.click(self.card_parent_link())
+        b.wait_visible(self.card_row("LVM2 volume group", name=disk))
+        self.click_card_dropdown("LVM2 volume group", "Delete group")
+        self.dialog_wait_open()
+        b.wait_text("#dialog td[data-label='Location']", "/")
+        self.dialog_apply()
+        self.dialog_wait_close()
+        m.execute("! findmnt --fstab -n /sysroot")
+
+        # Back to the beginning
+        b.wait_visible(self.card("Storage"))
+        b.wait(lambda: b.call_js_func('ph_count', self.card("Storage") + " tbody tr") == 1)
+        b.wait_not_present(self.card_row("Storage", location="/"))
```

Comment:
There are a lot of special cases/code changes in the stratis code. Could this be covered as well here? It needs a loop device instead of a RAM disk, but you can either replace it or just add a second one.